src/Symfony/Abstraction/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "symfony/abstractions",
+    "type": "library",
+    "description": "A set of domain specific abstractions maintained by the Symfony community",
+    "keywords": ["abstractions", "contracts", "decoupling", "interoperability"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "psr/cache": "~1.0",
+        "psr/container": "^1.0",
+        "psr/link": "^1.0",
+        "psr/log": "~1.0",
+        "psr/simple-cache": "^1.0"
+    },
+    "require-dev": {
+        "cache/integration-tests": "dev-master"
+    },
+    "autoload": {
+        "psr-4": { "Symfony\\Abstraction\\": "" },
+        "exclude-from-classmap": [
+            "**/Tests/"
+        ]
+    },
+    "minimum-stability": "dev",
+    "extra": {
```

Comment:
We don't use it anywhere yet. I can add it, but would it do anything actually? I feel like not, because we update these by hand (at least when merging), so that I don't think it will work...

Reply:
Many people learn to code by reading Symfony source code.
I think we should spread best practice and / or useful tips ;)